src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json

```diff
@@ -111,11 +112,12 @@
 					"http": "true",
 					"isVsix": "true",
 					"localization": "true",
-					"logging": "default",
+					"logging": "serilog",
```

Comment:
I'm not sure about setting serilog to be the default in the recommended template. Perhaps we should back off the template to using default instead of changing the wizard behavior

Reply:
This was always the default in Recommended, the TemplateWizard went rogue.